gatsby-node.js

```diff
@@ -104,6 +104,7 @@ exports.onCreatePage = ({ page, actions }) => {
         const is_production = process.env.GATSBY_ENV === 'production'
         const careers_regex = /^[a-z-]+\/careers\//g
```

Comment:
I think we can create one variable for e.g excluded page and check it from there